tests/test_views.txt

```diff
@@ -212,6 +212,7 @@ Tests for processes_rows()
 >>> processes1 = [
 ...     LocalRunningProcess(
 ...         pid="6239",
+...         xmin="1",
```

Comment:
Same here, we could use integers, no? (The same is probably true for `pid`, but that's unrelated and so cannot be changed now.)